src/Snapshot/InMemorySnapshotStore.php

```diff
@@ -1,37 +0,0 @@
-<?php
-/**
- * This file is part of the prooph/event-sourcing.
- * (c) 2014-2017 prooph software GmbH <contact@prooph.de>
- * (c) 2015-2017 Sascha-Oliver Prolic <saschaprolic@googlemail.com>
- *
- * For the full copyright and license information, please view the LICENSE
- * file that was distributed with this source code.
- */
-
-declare(strict_types=1);
-
-namespace Prooph\EventSourcing\Snapshot;
-
-use Prooph\EventSourcing\Aggregate\AggregateType;
-
-final class InMemorySnapshotStore implements SnapshotStore
```

Comment:
no, this is simply part of the snapshot-store together with the interface. so for testing you can use an inmemory version.